Hardcore.lua

```diff
@@ -144,6 +144,52 @@ local ALERT_STYLES = {
 -- the big frame object for our addon
 local Hardcore = CreateFrame("Frame", "Hardcore", nil, "BackdropTemplate")
 Hardcore.ALERT_STYLES = ALERT_STYLES
+Hardcore.verify_shown = false
+
+
+--[[ Ace3 config and imports ]]--
+
+Hardcore.Ace3 = LibStub("AceAddon-3.0"):NewAddon("Hardcore", "AceEvent-3.0", "AceConsole-3.0")
+local AceGUI = LibStub("AceGUI-3.0")
+
+local textStore
+
+function Hardcore:CloseVerifiedDialog(widget)
+	Hardcore.verify_shown = false
+	AceGUI:Release(widget)
+end
+
+function Hardcore:GetVerifiedDialog()
+
+	if Hardcore.verify_shown == true then
+		return
+	end
+	Hardcore.verify_shown = true
+	
+
+	local frame = AceGUI:Create("Frame")
+	frame:SetTitle("Classic Hardcore Verification")
+	frame:SetStatusText("Select All (Ctrl-A), Copy (Ctrl-C), Paste (Ctrl-V) this code to https://classichc.net/verify")
+	frame:SetCallback("OnClose", function(widget) Hardcore:CloseVerifiedDialog(widget) end)
+	frame:SetLayout("Flow")
+	
+	local editbox = AceGUI:Create("MultiLineEditBox")
+	editbox:SetLabel("Your validation code:")
+	local fakelongstring = Hardcore:GenerateVerificationString()
```

Comment:
This is just for demo sake right?

Reply:
You can merge to main or run my branch to test, but the feature is complete and done as a demo, and only fix for going live is removing the loop which doubles the export string (for the demo)

Reply:
Yeah thats what I meant - the for loop + fake string stuff. Sweet